lib/Parser/DocumentParser.php

```diff
@@ -109,6 +109,9 @@ final class DocumentParser
     /** @var string|null */
     private $listMarker = null;
 
+    /** @var array{name: string,body: string,offset: int}|null */
```

Comment:
We usually add a space after commas in such situations.

```suggestion
    /** @var array{name: string, body: string, offset: int}|null */
```